cart/views.py

```diff
@@ -61,80 +78,82 @@ def form_valid(self, form):
             colour=form.cleaned_data['colour'],
             size=form.cleaned_data['size'],
         )
-
-        if item_filter.exists():
-            item = item_filter.first()
-            item.quantity += int(form.cleaned_data['quantity'])
-            item.save()
-
-        else:
+        if not item_filter.exists():
```

Comment:
can separate the business logic, create a function in actions.py for example, to facilitate the presentation